src/commands.ts

```diff
@@ -0,0 +1,222 @@
+// -----------------------------------------------------------------------------
+// Utilities for command parsing
+// See chesster.js for command definitions.
+// -----------------------------------------------------------------------------
+import _ from 'lodash'
+
+const VARIABLE = /^\{(int)(text)|:[a-zA-Z]+\}$/
+const CHOICE_VARIABLE = /^\{([a-zA-Z0-9]+\|?)+\}$/
+const CONSTANT_VALUE = /^[a-zA-Z]+$/
+
+export class TextVariable {
+    typeName = 'text'
+    value: string
+    constructor(token: string) {
+        this.value = token
+    }
+}
+
+export class IntVariable {
```

Comment:
Is all this code used?

Reply:
IntVariable is, it's used in the availability commands.

Reply:
Were there other parts of this that you were concerned were unused?